### PR TITLE
Added csrf tag helper and framework for other helpers.

### PR DESCRIPTION
**Pull request by elorest**
### Description of the Change

Added a `helpers` folder at `src/amber/controller/helpers/`. Added `tag.cr` which currently only contains the `csrf_tag` helper.

### Why Should This Be In Core?

Csrf pipe is part of the core and this makes it easier for the end user to use it.

### CODE REVIEW

File: spec/amber/controller/base_spec.cr [hunk 1]
```diff
@@ -33,6 +33,26 @@ module Amber::Controller
 
         TestController.new(context).render_with_layout.should eq html_output
       end
+
+      it "renders a form with a csrf tag" do
+        request = HTTP::Request.new("GET", "/?test=test")
+        context = create_context(request)
+        html_output = <<-HTML
```
Comment (eliasjpr) [inline spec/amber/controller/base_spec.cr]: you can move this into a seed file and read the file contents to make the spec more cleaner.

File: src/amber/controller/base.cr [hunk 2]
```diff
@@ -1,11 +1,12 @@
 require "http"
-require "./*"
+require "./**"
```
Comment (eliasjpr) [inline src/amber/controller/base.cr]: 👍

File: src/amber/controller/helpers/tags.cr [hunk 3]
```diff
@@ -0,0 +1,7 @@
+module Amber::Controller::Helpers
```
Comment (eliasjpr) [inline src/amber/controller/helpers/tags.cr]: 👍

Comment (eliasjpr) [inline src/amber/controller/helpers/tags.cr]: I think the helper::tags should be in extensions or support since its adding functionality

File: spec/amber/controller/base_spec.cr (hunk 1 above)
Comment (elorest) [inline spec/amber/controller/base_spec.cr]: There are quite a few of these so a refactor of that should probably be a different PR.

File: src/amber/controller/helpers/tags.cr (hunk 3 above)
Comment (elorest) [inline src/amber/controller/helpers/tags.cr]: Perhaps, although it is directly related to controllers and about as useful as redirect or render. I think I'll merge it for now and when we start adding form helpers and stuff we can revisit this issue.
